docs/developer/app-store/apps/search.mdx

```diff
@@ -0,0 +1,267 @@
+---
+title: Search
+sidebar_position: 5
+---
+
+import { AppMetadata } from "/components/AppMetadata/AppMetadata.tsx";
+
+<AppMetadata
+  minSaleorVersion="3.10"
+  roadmapUrl="https://github.com/saleor/apps/labels/App%3A%20Search"
+  githubUrl="https://github.com/saleor/apps/tree/main/apps/search"
+/>
+
+## Introduction
+
+The application provides integration with [Algolia](https://www.algolia.com/) search engine. It allows you to index your products and search them using Algolia's API.
+
+## Features
+
+- Synchronize the whole catalog with Algolia
+- Update Algolia index on product changes
+
+## Assumptions & limitations
+
+- Initial synchronization has to be done manually in the App dashboard
```

Comment:
**polish**: I suggest dropping the "dashboard" bit, because it may be a bit confusing (app vs dashboard)

```suggestion
- Initial synchronization has to be done manually in the App
```